### PR TITLE
Morse code Decoder

### PR DESCRIPTION
**Pull request by alexpuente0**
On this PR we:

- Created an app to decode morse code messages (letters, words or sentences) using Ruby.
- Decoded the Secret message on the Exercise (A BOX FULL OF RUBIES).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,39 @@
...
 A simple app to decode morse code messages
+
+## Getting Started
+
+This repository includes files with plain SQL that can be used to recreate a database:
+
+- Use [schema.sql](./schema.sql) to create all tables.
+- Use [data.sql](./data.sql) to populate tables with sample data.
+- Check [queries.sql](./queries.sql) for examples of queries that can be run on a newly created database. **Important note: this file might include queries that make changes in the database (e.g., remove records). Use them responsibly!**
+
```
Comment (Gambit142) [inline README.md]: - [x] Kindly remove this section from your `README.md` file and include the necessary details for this section of your README.md file. Please include ways to clone this repository and run the ruby code in your `decoder.rb` file. 👍

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @alexpuente0 @aliveGUY 👋, 

Good job so far!

### Highlights
- All linters checks passed 💯 
- The methods return the correct result 💯 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: README.md (hunk 1 above)
Comment (alexpuente0) [reply README.md]: Already working on it, thanks!

**Review by mikethreels — APPROVED** [review]
Hi team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**